src/main/java/org/dita/dost/util/DitaClass.java

```diff
@@ -54,6 +58,7 @@ public DitaClass(final String cls) {
             sb.append(s).append(' ');
         }
         stringValue = sb.toString();
+        validDitaClass = (cls == null ? false : VALID_DITA_CLASS.matcher(stringValue).matches());
```

Comment:
No need to check for null, `cls` cannot be null and will throw a `NullPointerException` already on row 52 if it is.